app/views/layouts/application.html.erb

```diff
@@ -61,7 +61,20 @@
     </header>
 
     <div class="govuk-width-container">
-      <main class="govuk-main-wrapper " id="main-content" role="main">
+      <div class="govuk-phase-banner app-phase-banner--no-border govuk-!-display-none-print">
+        <p class="govuk-phase-banner__content">
+          <strong class="govuk-tag govuk-phase-banner__content__tag ">
+            Alpha
```

Comment:
Our favourite reference project has a Development banner, would that be more appropriate? I think eventually we will need a different banner depending on the hosting environment

Reply:
Truth be told I just copied text from the prototype :D 

I imagine this will likely be changed to Beta soon, and yeah, we might add environment into the mix when we deploy it and have multiple environments. I'm happy to switch it to development for now :)